conf/app.src.default

```diff
@@ -0,0 +1,5 @@
+SOURCE_URL=https://buildbot.libretro.com/stable/__UPSTREAM_VERSION__/emscripten/RetroArch.7z
+SOURCE_SUM=03166297e546e44755f1bacd8a4bbc90b58fae746864c3f67eaadd8c814906b2
```

Comment:
Eeeeh it works I guess, but you'll end up having to update the checksum each time the version changes anyway soooooo ? :thinking:

Reply:
yep, but I'm tired to update the version number at different places as I always forget one, so I tried to "centralized" it a bit :)
The things is: it worked on my local test but not on the package_check, so I obviously made some stupid mistake somewhere